packages/ord-connect/src/components/SelectWalletModal/index.tsx

```diff
@@ -32,7 +35,7 @@ export function SelectWalletModal({
     publicKey,
   } = useOrdContext();
   const [errorMessage, setErrorMessage] = useState<string>("");
-  const isChromium = window.chrome;
+  const supportedDevice = !disableMobile || !isMobileDevice();
```

Comment:
`supportedDevice` sounds like this variable contains the actual device instead of a boolean flag

Reply:
Will change to `isSupportedDevice`